File: VisitBosnia/visit_bosnia_mobile/lib/model/forum/forum.dart
import '../city/city.dart';

class Forum {
  int? id;
  String? title;
  // Null? description;
  int? cityId;
  String? createdTime;
  City? city;

  Forum(
      {this.id,
      this.title,
      // this.description,
      this.cityId,
      this.createdTime,
      this.city});

  Forum.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    title = json['title'];
    // description = json['description'];
    cityId = json['cityId'];
    createdTime = json['createdTime'];
    city = json['city'] != null ? new City.fromJson(json['city']) : null;
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['title'] = this.title;
    // data['description'] = this.description;
    data['cityId'] = this.cityId;
    data['createdTime'] = this.createdTime;
    if (this.city != null) {
      data['city'] = this.city!.toJson();
    }
    return data;
  }
}

File: VisitBosnia/VisitBosnia.Model/Forum.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VisitBosnia.Model
{
    public class Forum
    {
        public int Id { get; set; }
        public string Title { get; set; } = null!;
        public string? Description { get; set; }
        public int CityId { get; set; }
        public DateTime CreatedTime { get; set; }
        public virtual City City { get; set; } = null!;
        
    }
}
